includes/widgets/rating.php

```diff
@@ -0,0 +1,161 @@
+<?php
+namespace Elementor;
+
+use Elementor\Icons_Manager;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Elementor (new) rating widget.
+ *
+ * @since 3.17.0
+ */
+class Widget_Rating extends Widget_Base {
+
+	public function get_name() {
+		return 'rating';
+	}
+
+	public function get_title() {
+		return esc_html__( 'Rating', 'elementor' );
+	}
+
+	public function get_icon() {
+		return 'eicon-rating';
+	}
+
+	public function get_keywords() {
+		return [ 'star', 'rating', 'review', 'score', 'scale' ];
+	}
+
+	protected function register_controls() {
+		$this->start_controls_section(
+			'section_rating',
+			[
+				'label' => esc_html__( 'Rating', 'elementor' ),
+			]
+		);
+
+		$this->add_control(
+			'rating_maximum_value',
+			[
+				'label' => esc_html__( 'Rating Scale', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'range' => [
+					'px' => [
+						'min' => 1,
+						'max' => 10,
+					],
+				],
+				'step' => 1,
+				'default' => [
+					'size' => '5',
+				],
+			]
+		);
+
+		$this->add_control(
+			'rating_value',
+			[
+				'label' => esc_html__( 'Rating', 'elementor' ),
+				'type' => Controls_Manager::NUMBER,
+				'min' => 0,
+				'step' => 0.01,
+				'dynamic' => [
+					'active' => true,
+				],
+			]
```

Comment:
My bad, I had missed that.

Should we use a default value of 5 or should we use a placeholder value and set the default color with CSS?
I know last year we always tried to avoid default values in the controls.

What do you think? I will ask Doron on Slack about this as well.

Reply:
> Should we use a default value of 5 or should we use a placeholder value and set the default color with CSS?
> I know last year we always tried to avoid default values in the controls.

I think it was because we prioritize the ability to have default values as scss variables.
But I think that we need to use the default value here. especially if we want to be seen in the control UI.

Reply:
@davseve I have added an initial value inside the PHP and I have added a placeholder value. What do you think?